lib/code_fund_web/controllers/ad_serve_controller.ex

```diff
@@ -15,31 +15,39 @@ defmodule CodeFundWeb.AdServeController do
     cond do
       template == nil ->
         templates = Templates.list_templates()
-        template_slugs = Enum.map(templates, fn c -> c.slug end)
+
+        available_templates =
+          Enum.map(templates, fn c -> c.slug end)
+          |> Enum.join("|")
 
         conn
```

Comment:
These `conn` returns are basically the same code and the nested `cond` statements makes it a little hard to read, can it be restructured as a `with` clause?

Reply:
If you want to help me refactor this, sure 😄

Reply:
Sure I'll hop on a pairing session if you want.